src/ctap2/commands/mod.rs

```diff
@@ -103,13 +103,14 @@ pub(crate) trait PinAuthCommand {
         }
 
         let client_data_hash = self.client_data_hash();
-        let pin_auth = match calculate_pin_auth(dev, &client_data_hash, &self.pin()) {
-            Ok(pin_auth) => pin_auth,
-            Err(e) => {
-                return Err(repackage_pin_errors(dev, e));
-            }
-        };
-        self.set_pin_auth(pin_auth, Some(1)); // TODO(MS): Currently, we only support version 1
+        let (pin_auth, pin_auth_protocol) =
+            match calculate_pin_auth(dev, &client_data_hash, &self.pin()) {
```

Comment:
nit: I think `.map_err(|e| repackage_pin_errors(dev, e))?` is more idiomatic here.